mountpoint-s3/src/data_cache/express_data_cache.rs

```diff
@@ -72,6 +72,30 @@ where
             bucket_name: bucket_name.to_owned(),
         }
     }
+
+    pub async fn verify_cache_valid(&self) -> ObjectClientResult<(), PutObjectError, Client::ClientError> {
```

Comment:
A docstring explaining what check is done in this method and why is that sufficient? (e.g. what is `storage_class("EXPRESS_ONEZONE".to_string())` and why we don't do a `GetObject` check)